frontend/src/pages/modelServing/screens/const.ts

```diff
@@ -56,3 +56,19 @@ export const STORAGE_KEYS_REQUIRED: STORAGE_KEYS[] = [
   STORAGE_KEYS.SECRET_ACCESS_KEY,
   STORAGE_KEYS.S3_ENDPOINT,
 ];
+
+// unit: seconds
+export const TimeframeTime: TimeframeTimeType = {
+  [TimeframeTitle.FIVE_MINUTES]: 5 * 60,
+  [TimeframeTitle.ONE_HOUR]: 60 * 60,
+  [TimeframeTitle.ONE_DAY]: 24 * 60 * 60,
+  [TimeframeTitle.ONE_WEEK]: 7 * 24 * 60 * 60,
+};
+
+// make sure we always get ~300 data points
```

Comment:
I don't understand this comment -- if these are steps (x axis points) why does ~300 mean something?

Reply:
Oh, it's just because I think 300 is a proper number which is not much not less, can show you enough data but will not make it too choppy. and I calculated, if the step for 5 minutes timeframe is 1 second, there will be 300 seconds(data points), so I want to make sure all the timeframe have 300 data points.

Reply:
Ooooooh, interesting.